.babelrc

```diff
@@ -11,13 +11,8 @@
     ["@babel/typescript", { "allowNamespaces": true }]
   ],
   "plugins": [
-    "@babel/plugin-transform-modules-commonjs",
     ["@babel/plugin-proposal-decorators", { "legacy": true }],
-    "@babel/plugin-proposal-class-properties",
-    "@babel/proposal-object-rest-spread",
-    "babel-plugin-styled-components",
-    "@babel/plugin-syntax-dynamic-import",
-    "babel-plugin-lodash"
+    "babel-plugin-styled-components"
```

Comment:
Is there anything else controlling babel plugins in the tree, or should the removed babel plugins also be removed from `package.json`/`yarn.lock`?